resource_lists/minecraft/resources.yaml

```diff
@@ -14273,6 +14273,15 @@ resources:
         recipe:
         - Oak Wood
         - Fuel
+     - output: 1
```

Comment:
This line is indented by 1 extra space breaking the yaml parser and causing build to crash. Ill pull this and fix it over the weekend unless you get to it first.